_rules/audio-is-live-213x3x.md

```diff
@@ -0,0 +1,123 @@
+---
+id: 213x3x
+name: '`audio` element plays live content'
+rule_type: atomic
+description: |
+  This rule checks the content played by an `audio` element is live
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Audio output
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+This rule applies to every `audio` element that is:
+
+- playing; or,
+- has a [play button][] that is [visible][] and [included in the accessibility tree][].
+
+## Expectation 
+
+For each test target, after [seeking][seek] the end of the target element's [media resource][] the [current playback position][] is not the end of the target element's [media resource][].
+
+## Assumptions
+
+_No assumptions._
+
+## Accessibility Support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- [Understanding SC 1.2.1:Audio-only and Video-only (Prerecorded)](https://www.w3.org/WAI/WCAG21/Understanding/audio-only-and-video-only-prerecorded)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `audio` element plays a live media resource by streaming the audio from the device's microphone.
```

Comment:
Also, have you tested that by "seeking" the end, to make sure this matches the expectation?
Wondering if the "seek" bit should be used also in examples descriptions 🤔

Reply:
I just found that MediaStream is not seekable! https://w3c.github.io/mediacapture-main/#mediastreams-in-media-elements 
I will add this to the expectation and the example's description.

Reply:
And I suppose this lands us again exactly where we were before... needing to find a live stream that we can use in the example 😞